### ISSUE TITLE
changing sort order field causes bugs to move around

### ISSUE THREAD

**Issue by willkg** [mythmon/edwin#62]
When I'm typing in the sort order field, it causes bugs to move around. The worst case scenario here is when I remove all the text in the text field--then the bug disappears. I have to hard-refresh the page for the bug to show up again.

To clarify, I'm literally just backspacing the text in the sort field for the bug and then the bug disappears--I'm not hitting the enter key or clicking on commit sort or anything like that.

**Comment by mythmon** [comment]
Yeah, this sucks. It probably isn't disappearing, by the way, but instead moving to the unsorted section of the page.

The reason for this is that the bugs are grouped into sorted/unsorted based on the sort order field, and the text box is directly linked to the sort order field, instead of having a layer of indirection, which would allow you to edit it without anything changing.

I've been procrastinating on this, because I expect some day @rehandalal will come by and fix #56, and we won't have to deal with this anymore anyways.

**Comment by willkg** [comment]
Actually, that's handy to know. It means I can push stuff out of the timeline that I'm no longer planning to work on.
